Remotion/Data/DomainObjects.UnitTests/Queries/Configuration/Loader/QueryDefinitionFileLoaderTest.cs

```diff
@@ -55,10 +55,14 @@ public void LoadQueryDefinitions ()
       var actualQueries = loader.LoadQueryDefinitions(GetFullScriptPath("QueriesForLoaderTest.xml"));
       var expectedQueries = new[]
                             {
-                                TestQueryFactory.CreateOrderQueryWithCustomCollectionType(StorageSettings),
-                                TestQueryFactory.CreateOrderQueryDefinitionWithObjectListOfOrder(StorageSettings),
-                                TestQueryFactory.CreateCustomerTypeQueryDefinition(StorageSettings),
-                                TestQueryFactory.CreateOrderSumQueryDefinition(StorageSettings)
+                                TestQueryFactory.CreateOrderQueryWithCustomCollectionTypeReadOnly(StorageSettings),
+                                TestQueryFactory.CreateOrderQueryDefinitionWithObjectListOfOrderReadOnly(StorageSettings),
+                                TestQueryFactory.CreateCustomerTypeQueryDefinitionReadOnly(StorageSettings),
+                                TestQueryFactory.CreateOrderSumQueryDefinitionReadOnly(StorageSettings),
+                                TestQueryFactory.CreateOrderQueryWithCustomCollectionTypeReadWrite(StorageSettings),
+                                TestQueryFactory.CreateOrderQueryDefinitionWithObjectListOfOrderReadWrite(StorageSettings),
+                                TestQueryFactory.CreateCustomerTypeQueryDefinitionReadWrite(StorageSettings),
+                                TestQueryFactory.CreateOrderSumQueryDefinitionReadWrite(StorageSettings),
```

Comment:
Here we want to add the custom-readonly and custom-readwrite queries and drop those that are redudant.

Reply:
I dont know which are redundant